core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -300,6 +300,9 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
                 }
                 // Strip debug symbols regardless, because the underlying JDK might contain them
                 objcopy("--strip-debug", finalPath.toString());
+            } else {
+                log.info("objcopy executable not found in PATH. Debug symbols will not be separated from executable!");
```

Comment:
Let's drop the "!" please.
And it probably needs to be a `warn`?

Reply:
Done